Add glossary entry for “stringifier” (#12038)
mdn/content: files/en-us/glossary/stringifier/index.md

---
title: Stringifier
slug: Glossary/Stringifier
tags:
  - CodingScripting
  - Glossary
  - Stringifier
---

> An {{Glossary("object", "object's")}} stringifier is any {{Glossary("attribute")}} or {{Glossary("method")}} that is defined to provide a textual representation for use when the object is used in situations where a {{Glossary("string")}} is expected e.g. `console.log(anOjbect)`. Objects that don't have any defined stringifer, are usually converted to their {{Glossary("JSON")}} representation in similar situations.

Reviewer: ```suggestion
An {{Glossary("object", "object's")}} stringifier is any {{Glossary("attribute")}} or {{Glossary("method")}} that is defined to provide a textual representation of the object for use in situations where a {{Glossary("string")}} is expected.
```
